src/form/password-input/PasswordInput.tsx

```diff
@@ -0,0 +1,81 @@
+import "./_password-input.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+
+import Input, {InputProps} from "../input/Input";
+import Button from "../../button/Button";
+
+export interface PasswordInputProps extends Omit<InputProps, "leftIcon" | "rightIcon"> {
```

Comment:
Let's convert to this, as far as I can see you're not overriding `InputProps` in here 

```suggestion
export interface PasswordInputProps extends Omit<InputProps, "leftIcon" | "rightIcon"> {
  hideIcon?: React.ReactNode;
  revealIcon?: React.ReactNode;
}
```